htop/Containerfile

```diff
@@ -0,0 +1,6 @@
+#Start from your base image. Here I'm using rhel 9.2 from the 4.14 image
+FROM registry.ci.openshift.org/ocp/4.14-2023-03-16-110647@sha256:cd299b2bf3cc98fb70907f152b4281633064fe33527b5d6a42ddc418ff00eec1
+#Install epel repo rpm(I'm using the rhel9 version), followed by htop
```

Comment:
I find use of the first person view in code comments to usually be a bit odd, because one wants to assocate it with a person but that context gets lost over time.  I find comments more readable if they're declarative; how about:

> Enable EPEL (more info at https://docs.fedoraproject.org/en-US/epel/ ) and install htop

Reply:
That makes sense! I updated the comment to reflect this.